cmd/utils.go

```diff
@@ -245,6 +250,35 @@ func DumpConfig(configPath, serviceName string, useDefault bool) error {
 	return nil
 }
 
+func checkVersion(config map[string]interface{}) error {
+	if config == nil {
+		return nil
+	}
+	configVersion, ok := config["version"]
+	if !ok {
+		return errors.New("config hasn't version key")
+	}
+	versionValue, ok := configVersion.(string)
+	if !ok {
+		return errors.New("value of version is not string")
+	}
+
+	version, err := utils.ParseVersion(versionValue)
+	if err != nil {
+		return err
+	}
+
+	serverVersion, err := utils.GetParsedVersion()
+	if err != nil {
+		return err
+	}
+
+	if serverVersion.CompareOnly(utils.MajorFlag|utils.MinorFlag, version) != utils.Equal {
+		return errors.New("config version is outdated")
```

Comment:
Another possibility is that we're trying to use a new config with an old Acra version. I'd rephrase this message into something like

```suggestion
		return errors.New("config version is not supported")
```

It may be also useful to include the versions in the error so that it's clear what config version we have in the file and what is the version of Acra.

Reply:
agree, makes sense to rename

Reply:
okay, will add this